src/patternfly/components/Check/check.scss

```diff
@@ -1,37 +1,34 @@
 @import "../../patternfly-utilities";
 
 .pf-c-check {
-  --pf-c-check__label--PaddingLeft: var(--pf-global--spacer--sm);
-  --pf-c-check__label--disabled--color: var(--pf-global--disabled-color--100);
+  --pf-c-check--GridGap: var(--pf-global--spacer--sm);
+  --pf-c-check__label--disabled--Color: var(--pf-global--disabled-color--100);
   --pf-c-check__label--Color: var(--pf-global--Color--100);
   --pf-c-check__label--FontWeight: var(--pf-global--FontWeight--normal);
   --pf-c-check__label--FontSize: var(--pf-global--FontSize--md);
   --pf-c-check__label--LineHeight: var(--pf-global--LineHeight--sm);
 
-  .pf-c-check__label {
-    display: inline-block;
-    font-size: var(--pf-c-check__label--FontSize);
-    font-weight: var(--pf-c-check__label--FontWeight);
-    line-height: var(--pf-c-check__label--LineHeight);
-    color: var(--pf-c-check__label--Color);
-  }
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-gap: var(--pf-c-check--GridGap);
+  align-items: baseline;
+}
```

Comment:
This will fix an alignment issue in FF
```suggestion
justify-items: start;
}
```

Reply:
Check it out now. Looks like `align-items: baseline` produces inconsistent vertical alignment, but `align-items: center` produces consistent alignment, although it's off center. Using a negative margin on the input (copying [Bootstrap](https://getbootstrap.com/docs/4.0/components/forms/#checkboxes-and-radios)) seems to do the trick. What do you think?

Reply:
This is interesting. I investigated in Firefox, it looks like the checkbox aligns with the x-height of the text 
<img width="268" alt="screen shot 2018-12-06 at 9 39 33 am" src="https://user-images.githubusercontent.com/20118816/49591122-900da480-f93b-11e8-8ad2-59a2fe0de0cf.png">

Reply:
If Bootstrap is using that method I think its okay to move forward. Except for the fact that its a hardcoded value and  so any change in font size should shift the position again. Could you incorporate the line-height of the check__label?

Reply:
I tested with and without the line-height, and didn't see any difference. The `-3px` is a rem value, and the font size is, too, so I tested changing the label's font-size and the input is aligned as well as I would expect. FYI, I tested this using grid/align-items: center;, flex/align-items: center;, and inline-block/verticle-align: middle, and it all produced consistent results in Chrome, FF, and Edge, so I stuck with grid since we can use `grid-gap` to control the horizontal spacing. It's also worth noting that I bet we'll end up changing these checkboxes to accommodate for a larger target area, so this may change down the road, too. What do you think?

Reply:
Okay nice that sounds good to me!